styles/fonts/DP Sans Mono License.txt

```diff
@@ -1,38 +1,42 @@
-Copyright (c) 2003 by Distributed Proofreaders. All Rights Reserved.
+Fonts are (c) Bitstream (see below). Changes made by DejaVu and by
+Distributed Proofreaders are in public domain.
 
-Copyright (c) 2003 by Bitstream, Inc. All Rights Reserved.
-Bitstream Vera is a trademark of Bitstream, Inc.
+Bitstream Vera Fonts Copyright
+-------------------------------
 
-Permission is hereby granted, free of charge, to any person obtaining
-a copy of the fonts accompanying this license (“Fonts”) and associated
-documentation files (the “Font Software”), to reproduce and distribute
-the Font Software, including without limitation the rights to use,
-copy, merge, publish, distribute, and/or sell copies of the Font
-Software, and to permit persons to whom the Font Software is furnished
-to do so, subject to the following conditions:
+Copyright (c) 2003 by Bitstream, Inc. All Rights Reserved. Bitstream
+Vera is a trademark of Bitstream, Inc.
+
+Permission is hereby granted, free of charge, to any person obtaining a
+copy of the fonts accompanying this license ("Fonts") and associated
+documentation files (the "Font Software"), to reproduce and distribute
+the Font Software, including without limitation the rights to use, copy,
+merge, publish, distribute, and/or sell copies of the Font Software, and
+to permit persons to whom the Font Software is furnished to do so,
+subject to the following conditions:
 
 The above copyright and trademark notices and this permission notice
 shall be included in all copies of one or more of the Font Software
 typefaces.
 
 The Font Software may be modified, altered, or added to, and in
 particular the designs of glyphs or characters in the Fonts may be
-modified and additional glyphs or characters may be added to the
-Fonts, only if the fonts are renamed to names not containing either
-the words “Bitstream” or the word “Vera”.
+modified and additional glyphs or or characters may be added to the
```

Comment:
"or or"?

Reply:
I fixed this. It's still wrong in the font but I'm not too worried about it.